daisy_workflows/image_build/debian/build.py

```diff
@@ -123,6 +124,50 @@ def main():
   synopsis_dest = os.path.join(outs_path, 'synopsis.json')
   utils.UploadFile('/tmp/synopsis.json', synopsis_dest)
 
+  # Create and upload metadata of the image and packages
+  logging.info('Creating image metadata.')
+  image = {
+      "family": "debian-9",
+      "name": re.match("debian-9-[\\w-]*", image_dest),
+      "version": "stretch",
+      "location": image_dest,
+      "release_date": datetime.now(timezone.utc),
+      "packages": []
+  }
+  # Read list of guest package
+  with open("guest_package") as f:
+    guest_packages = f.read().splitlines()
+
+  for package in guest_packages:
+    cmd = "dpkg-query -W --showformat '${Package}:${Version}:${Git}'" \
```

Comment:
given the version has a colon in it, you might use a different separator

Reply:
maybe use newline as you did for EL?

Reply:
Let's unified with "\n"